.go-env.sh

```diff
@@ -32,8 +45,17 @@ args=(
 	--env DOCKERHUB_PUBLIC_PROXY
 	--env DOCKERHUB_PUBLIC_PROXY_HOST
 )
+
+if [ -z "$windowsContainers" ]; then
+	user="$(id -u):$(id -g)"
```

Comment:
```suggestion
	user="$(id -u)"
	user+=":$(id -g)"
```

While we're here, we might as well improve the error handling here :eyes: